diff -Naru gnuplot-5.4.0_orig/Makefile.in gnuplot-5.4.0/Makefile.in
--- gnuplot-5.4.0_orig/Makefile.in	2020-07-13 12:38:13.000000000 +0900
+++ gnuplot-5.4.0/Makefile.in	2020-11-02 18:16:44.561619657 +0900
@@ -352,7 +352,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = config m4 term src docs man demo share
+SUBDIRS = config m4 term src man share
 EXTRA_DIST = BUGS Copyright FAQ.pdf INSTALL INSTALL.gnu \
 PATCHLEVEL PGPKEYS README RELEASE_NOTES \
 configure.vms win
